apps/nextjs-website/src/components/templates/GitBookTemplate/GitBookTemplate.tsx

```diff
@@ -7,6 +7,7 @@ import ProductBreadcrumbs from '@/components/atoms/ProductBreadcrumbs/ProductBre
 import { FragmentProvider } from '@/components/organisms/FragmentProvider/FragmentProvider';
 import GitBookContent from '@/components/organisms/GitBookContent/GitBookContent';
 import GuideInPageMenu from '@/components/organisms/GuideInPageMenu/GuideInPageMenu';
+import { useChatbot } from '@/helpers/chatbot.helper';
```

Comment:
I something missing in this file or is just a useless import :thinking:  ? If there isn't something missing i would delete this line :wink: 
```suggestion
```